### PR TITLE
Update ansibleVars

### PR DESCRIPTION
**Pull request by fao89**
ansibleVars is not string anymore:
https://github.com/openstack-k8s-operators/dataplane-operator/pull/319
https://github.com/openstack-k8s-operators/install_yamls/pull/442

### CODE REVIEW

File: tests/roles/dataplane_adoption/tasks/main.yaml
Comment (marios) [inline tests/roles/dataplane_adoption/tasks/main.yaml:143]: so does it mean we can expect the currently running job in the periodic line to fail until we merge this now?

right now the latest run in history is from yesterday  https://review.rdoproject.org/zuul/builds?job_name=periodic-centos-9-standalone-to-crc-data-plane-adoption-antelope

but currently in the queue https://review.rdoproject.org/zuul/status

periodic-centos-9-standalone-to-crc-data-plane-adoption-antelope waiting

**Review by marios — APPROVED** [review]
lgtm and tests green on this

**Review by cescgina — APPROVED** [review]
lgtm, I picked this up in a testproject run and it fixes the job: https://review.rdoproject.org/zuul/build/451b9123d71a4124aa3bdbfd55499a3d

**Comment by fao89** [comment]
merging as it got 2 approvals

**Pull request merged**
